src/main/resources/application.yaml

```diff
@@ -16,9 +16,6 @@ spring:
   freemarker:
     # If Spring should check it's default template location (always false since we handle our own Freemarker templates)
     checkTemplateLocation: false
-logging:
-  # Logging path for Spring, which should be the same as the Deployer logging path
-  path: ${deployer.main.logging.folderPath}
```

Comment:
This is already done by the `application.yaml` in the `craftercms` config: https://github.com/craftercms/craftercms/blob/develop/resources/deployer/config/application.yaml. Maybe it's better to move all the sys env properties from that file to here at this point.